### DISCUSSION TITLE
计算机视觉 - 视频理解 的视频会上传吗？

### DISCUSSION THREAD

**Discussion by Leozyc-waseda** [mli/paper-reading#251] [Q&A]
看到[计算机视觉 - 视频理解](https://github.com/mli/paper-reading#%E8%AE%A1%E7%AE%97%E6%9C%BA%E8%A7%86%E8%A7%89---%E8%A7%86%E9%A2%91%E7%90%86%E8%A7%A3)都是打了绿色的勾勾的，想问问什么时候会放视频出来啊，特别感兴趣。

**Comment by stduent-vehicle** [comment]
> 看到[计算机视觉 - 视频理解](https://github.com/mli/paper-reading#%E8%AE%A1%E7%AE%97%E6%9C%BA%E8%A7%86%E8%A7%89---%E8%A7%86%E9%A2%91%E7%90%86%E8%A7%A3)都是打了绿色的勾勾的，想问问什么时候会放视频出来啊，特别感兴趣。

https://www.bilibili.com/video/BV1fL4y157yA/?vd_source=2305730152e6c9a557978924d0063b1a

**Comment by stduent-vehicle** [comment]
> 看到[计算机视觉 - 视频理解](https://github.com/mli/paper-reading#%E8%AE%A1%E7%AE%97%E6%9C%BA%E8%A7%86%E8%A7%89---%E8%A7%86%E9%A2%91%E7%90%86%E8%A7%A3)都是打了绿色的勾勾的，想问问什么时候会放视频出来啊，特别感兴趣。

@Leozyc-waseda
